### DISCUSSION TITLE
2.3.0版本安装有问题

### DISCUSSION THREAD

**Discussion by hchero** [lyswhut/lx-music-desktop#1436] [Q&A]
window11安装后只有一个卸载程序，其他文件都没有了

**Comment by Eastinliang** [comment]
> After installing windows 11, there is only one uninstaller, and the other files are gone

https://github.com/lyswhut/lx-music-desktop/releases/download/v2.3.0/lx-music-desktop-v2.3.0-win_x64-green.7z

**Comment by aectn** [comment]
用剩下的那个卸载程序完全卸载之后程序安装
